README.md

```diff
@@ -55,68 +55,73 @@ __For Mac users__: Follow the step-by-step instruction below (we are working on
 
 4. Install `bundler`:
 
-  ```
+  ```sh
   gem install bundler
   rbenv rehash
   ```
 
-5. `cd` into `bin` and install the required gems:
+5. Install the required gems:
 
   ```sh
+  cd bin
   bundle install
   ```
-  You need to have [MySQL installed](http://dev.mysql.com/downloads/mysql/) before hand.
   
   As Gems get updated, this can have problems. Check the [FAQ] (https://github.com/autolab/Autolab/wiki/FAQ).
 
 
-6.  Configure your database next. You need to fill the `username` and `password` fields on 		`config/database.yml.template` and rename it to `config/database.yml`. Depending on how you installed MySQL and which platform you're running on, you might have to change the database socket setting in this file. If you're having trouble, look at the [FAQ] (https://github.com/autolab/Autolab/wiki/FAQ)
+6.  Configure your SQLite database with the following command.
 
-7. Set up initializer for Devise Auth systems with a unique key.
-   
-   ```console
-   cp config/initializers/devise.rb.template config/initializers/devise.rb
+   ```sh
+   cd ../config
+   sed -n '12,16p' < database.yml.template | sed -e "s/#//g" >> database.yml
```

Comment:
It takes the sqllite part from the database template and makes the new database.yml file from the extracted lines.